cantabular/queries.go

```diff
@@ -279,20 +285,15 @@ func (c *Client) queryUnmarshal(ctx context.Context, graphQLQuery string, data Q
 		return dperrors.New(
 			fmt.Errorf("failed to read response body: %s", err),
 			res.StatusCode,
-			log.Data{
-				"url": url,
-			},
+			logData,
 		)
 	}
 
 	if err := json.Unmarshal(b, v); err != nil {
 		return dperrors.New(
```

Comment:
Notice this error has the extra responsebody.String() added, you can add it to the logData here